Source/Libs/GLibSharp/GType.cs

```diff
@@ -238,7 +239,7 @@ public static Type LookupType (IntPtr typeid)
 					break;
 			}
 
-			if (result == null) {
+			if (result == null && RuntimeFeature.IsDynamicCodeSupported) {
```

Comment:
The statement few lines down will always return false due to the assembly names 😅  
 
```cs
string asm_name = ns.ToLower ().Replace ('.', '-') + "-sharp";

if (ref_name.Name != asm_name)
``` 

Just adding this comment for myself, nothing for you to fix here 😄

Reply:
That’s code review :) every side can participate